pytransform3d/test/test_batch_rotations.py

```diff
@@ -42,6 +43,34 @@ def test_norm_vectors_zero():
     assert_array_almost_equal(V_unit, V)
 
 
+def test_norm_axis_angles():
+    rng = np.random.default_rng(843)
+    A = np.hstack((
+        pbr.norm_vectors(rng.standard_normal(size=(10, 3))),
+        np.pi + rng.uniform(size=(10, 1))
+    ))
+
+    # 1D
+    assert_array_almost_equal(
+        pbr.norm_axis_angles(A[0]), pr.norm_axis_angle(A[0]))
+
+    assert_array_almost_equal(
+        pbr.norm_axis_angles(np.array([[[0.0, 0.0, 0.0, np.pi]]]))[0, 0],
+        pr.norm_axis_angle(np.array([0.0, 0.0, 0.0, np.pi])))
+
+    # 2D
+    A_norm = pbr.norm_axis_angles(A)
+    for a, a_norm in zip(A, A_norm):
+        assert pytest.approx(a[3]) != a_norm[3]
```

Comment:
Consider leaving a comment why is it guaranteed here? A naive reader (like me :D) may assume only the rotation vector is normalized but the angle is unchanged?

Reply:
Good point. Doesn't make sense to normalize the axis when checking this function. I changed that to

```python
    # create a batch of unnormalized axis-angle instances
    n_rotations = 10
    A_unnormalized = np.hstack((
        rng.standard_normal(size=(n_rotations, 3)),
        np.pi + rng.uniform(size=(n_rotations, 1))
    ))
```

Then this loop becomes

```python
    for a_unnormalized, a_norm in zip(A_unnormalized, A_norm):
        # Make sure that the unnormalized version was in fact not normalized.
        assert pytest.approx(a_unnormalized[3]) != a_norm[3]
        assert pytest.approx(np.linalg.norm(a_unnormalized[:3])) != 1.0
        assert_array_almost_equal(a_norm, pr.norm_axis_angle(a_unnormalized))
```